juriscraper/opinions/united_states/state/ill.py

```diff
@@ -42,20 +48,27 @@ def _process_html(self) -> None:
         """
         for row in self.html.xpath("//table[@id='ctl04_gvDecisions']/tr"):
             cells = row.xpath(".//td")
-
             # Don't parse pagination rows or headers or footers
             if len(cells) != 7 or row.xpath(".//table"):
                 continue
-
-            name = get_row_column_text(row, 1)
-            date = get_row_column_text(row, 3)
-            citation = get_row_column_text(row, 2)
             try:
                 url = get_row_column_links(row, 1)
             except IndexError:
+                logger.info(
+                    f"Opinion without URL to file: '{str(tostring(row))}'"
+                )
                 # If the opinion file's information is missing (as with
                 # links to withdrawn opinions), skip record
```

Comment:
Im not sure we would want to just log the entire row as HTML.

Reply:
It's useful to detect changes in the structure of the table with the opinions lists and, for this particular validation, if the link to the opinion file is still being published in this column.